server/ai_session.go

```diff
@@ -138,6 +142,7 @@ func (pool *AISessionPool) Clear(newSessions []*BroadcastSession) {
 		}
 		if toRemove {
 			pool.selector.Remove(pool.sessMap[k])
+			pool.inUseSess = removeSessionFromList(pool.inUseSess, pool.sessMap[k])
```

Comment:
This is the actual fix of the bug introduced in https://github.com/livepeer/go-livepeer/pull/3493